packages/vue/src/components/app-footer.vue

```diff
@@ -1,21 +1,22 @@
 <template>
-  <div class="content-block">
-    <footer class="footer">
-      Copyright © {{ new Date().getFullYear() }} Developer Express Inc.
-    </footer>
-  </div>
+  <footer class="footer">
+    Copyright © {{ new Date().getFullYear() }} <br/> Developer Express Inc.
```

Comment:
There is many warnings about `Disallow self-closing on HTML void elements (<br/>)` . Could be it fixed?

Reply:
Fixed